debian/ola.udev

```diff
@@ -4,6 +4,9 @@ SUBSYSTEM=="usb|usb_device", ACTION=="add", ATTRS{idVendor}=="0403", ATTRS{idPro
 # udev rules for the anyma dmx device
 SUBSYSTEM=="usb|usb_device", ACTION=="add", ATTRS{idVendor}=="16c0", ATTRS{idProduct}=="05dc", GROUP="plugdev"
 
+# udev rules for the avldiy dmx device
```

Comment:
Can you also open a PR here to patch the pi file: https://github.com/OpenLightingProject/raspberrypi/blob/master/etc/udev/rules.d/10-local.rules